scripts/build-config-fixtures.js

```diff
@@ -0,0 +1,52 @@
+import yazl from 'yazl'
+import fs from 'node:fs/promises'
+import { createWriteStream } from 'node:fs'
+
+const CONFIG_FIXTURES_PATH = new URL(
+  '../tests/fixtures/config',
+  import.meta.url
+).pathname
+const dir = await fs.readdir(CONFIG_FIXTURES_PATH)
+console.log('zipping config fixtures')
+for (const fileOrFolder of dir) {
+  const p = `${CONFIG_FIXTURES_PATH}/${fileOrFolder}`
```

Comment:
This might not work on Windows. Can we use `path.join` instead?

```js
import { join } from 'node:path';
// ...
```

```suggestion
  const p = join(CONFIG_FIXTURES_PATH, fileOrFolder)
```